.github/workflows/regression.yml

```diff
@@ -16,7 +16,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.6', '3.7', '3.8', '3.10', '3.11']
+    # 3.9 no longer works because pint does not support it https://github.com/hgrecco/pint/issues/1974#issuecomment-2172693837
```

Comment:
If I understand correctly, the error you are seeing in the CI is because it is using pint 0.23 with numpy 2.0. So wouldn't the better option be to have omas specifiy that on python version 3.9 and older you only support numpy 2.0.

Or deprecate all versions before 3.10. But supporting 3.6-3.8 and dropping 3.9 seems arbitrary given that it's an issue of dependencies being mixed incorrectly, and not an issue with python 3.9

Reply:
Thanks for calling out my laziness here. That is definitely a better solution!